### ISSUE TITLE
USE statement tokenising can sometimes result in mismatched scopes

### ISSUE THREAD

**Issue by JustBlackBird** [squizlabs/PHP_CodeSniffer#592]
I'm using v2.3.2 of PHP Code Sniffer and I have `test.php` file with the following contents:
```php
<?php

use Vendor\Test\FooBar;

function test($val = null)
{
    if ($val) {
        switch ($val) {
            case '1':
                $result = 'one';
                break;
            case '2':
                $result = 'two';
                break;
            default:
                $result = 'cannot count';
        }
    } else {
        $result = 'none';
    }

    return $result;
}

```
and when I run PHPCS using the following command:
```shell
$ ./vendor/bin/phpcs --standard="PSR2" test.php
```
it outputs error report:
```
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 22 | ERROR | [x] Terminating statement must be indented to the same
    |       |     level as the CASE body
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

As for me it's unexpected behavior because the PHP file follows PSR-2 standard. The most interesting thing is if I remove
```php
use Vendor\Test\FooBar;
```
the error disappeared.

Is this a bug or I've missed something?

**Comment by gsherwood** [comment]
Thanks for reporting. I've pushed a fix.

**Comment by JustBlackBird** [comment]
Thank you, it works!
